.gitignore

```diff
@@ -9,4 +9,6 @@
 /uploads/
 node_modules/
 wordpress-coding-standards
-php-codesniffer
\ No newline at end of file
+php-codesniffer
+plugins/wordpress-importer/
+php.ini
```

Comment:
php.ini file is Apache web server configuration file and should never be found in wp-content folder. But we can't assume that it will be because we'll end up ignoring complete Apache file tree.